diff --git a/matlab/CSXGeomPlot.m b/matlab/CSXGeomPlot.m
index 21d3ef8..37427a1 100644
--- a/matlab/CSXGeomPlot.m
+++ b/matlab/CSXGeomPlot.m
@@ -13,18 +13,10 @@ if nargin < 1
     error 'specify the xml file to open'
 end
 
-filename = mfilename('fullpath');
-dir = fileparts( filename );
-AppCSXCAD_Path = [dir filesep '../../AppCSXCAD' filesep];
-   
-if (~exist(AppCSXCAD_Path,'dir'))
-    AppCSXCAD_Path = [dir filesep '..' filesep]  
-end
-
 if isunix
-	AppCSXCAD_Path = [AppCSXCAD_Path 'AppCSXCAD.sh'];
+	AppCSXCAD_Path = 'AppCSXCAD.sh';
 else
-	AppCSXCAD_Path = [AppCSXCAD_Path 'AppCSXCAD'];
+	AppCSXCAD_Path = 'AppCSXCAD';
 end
 
 command = [AppCSXCAD_Path ' --disableEdit ' CSX_filename];
